site-specific/cancer-care-associates/config.toml

```diff
@@ -1,20 +1,40 @@
 version = 1
 
-# TODO: For config v2 scope 'data_methods' under an 'mudensity' config
+# TODO [v2]: Scope 'data_methods', 'output', and 'gamma' under a 'metersetmap'
+# config
 [data_methods]
 available = ["monaco", "dicom", "icom", "trf", "mosaiq"]
 default_reference = "monaco"
 default_evaluation = "icom"
 
+[output]
+png_directory = '\\pdc\PExIT\Physics\Patient Specific Logfile Fluence'
+
+[gamma]
+dose_percent_threshold = 2
+distance_mm_threshold = 0.5
+local_gamma = true
+quiet = true
+max_gamma = 5
+lower_percent_dose_cutoff = 20
+
+# TODO [v2]: Consider changing from:
+#     {"site": [{"name": "rccc", ...}, {"name": "nbcc", ...}]}
+# to:
+#     {"site": {"rccc": { ... }, "nbcc": { ... } }}
+# The question here, 'is order important'? And maybe order preservation is
+# helpful, in which case it should remain scoped within a list as opposed to
+# a dictionary.
 [[site]]
 name = "rccc"
 
-    # TODO: For config v2 change 'export-directories' to just 'directories'
+    # TODO [v2]: Change 'export-directories' to just 'directories'
```

Comment:
best to leave inputs and outputs separate.
however, is this a set of directories to which pymedphys exports, or is this a directory to which other applications export, and is therefore considered an import directory for pymedphys?
Will pymedphys have places where it wants to export something as inputs to these applications/protocols?

Reply:
In the case of input/output I was thinking of making that clear within the directories themselves if that was appropriate. Thinking of these config files more as "root directories" for a given scenario. Each application can then run away and make well defined sub directories relative to these roots if need be. What do you think regarding that?

The `icom = '\\rccc-physicssvr\iComLogFiles'` entry is being used within my `wlutz-arc` branch to point to where the iCom log files are being stored. However, it is not unreasonable that the very iCom listener that is listening to the iCom stream and saving these log files (also PyMedPhys) could pull from this config option. So in one sense, for the `pymedphys icom listen` CLI this would be an export directory, for the WLutz Arc GUI it would be an input. ... the concepts of export/import input/output tend to be blurred.

Looking for any further input on this you might have. Thoughts?